src/js/api/config.js

```diff
@@ -84,6 +85,13 @@ const Config = function(options, persisted) {
     config.language = language;
     config.intl = intl;
 
+    // If autoPause is configured with an empty block,
+    // default autoPause.viewability to true.
+    let autoPause = allOptions.autoPause;
+    if (autoPause) {
+        config.autoPause.viewability = ('viewability' in autoPause) ? !!autoPause.viewability : true;
```

Comment:
Why do you need to use `in`? That operator can be a bit tricky because it also checks the prototype chain.

Reply:
@johnBartos I was originally using `hasOwnProperty` but Rob suggested this way.

Reply:
ok cool